index.html

```diff
@@ -270,87 +291,285 @@
 It will continue to evolve and the task force will issue updates as often as needed. At the
 conclusion of the task force, the TAG intends to adopt this document as a Finding.
 
+<aside class="note">
+
+Privacy is covered by legal frameworks and this document recognises that existing data
+protection laws take precedence for legal matters. However, because the Web is global, we
+benefit from having shared concepts to guide its evolution as a system built for the people using it
+([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
+can hopefully help all the Web's participants in different legal regimes. Our shared
+understanding is that the law is a floor, not a ceiling.
+
+</aside>
+
 </section>
 
-# Introduction {#intro}
+# Understanding Privacy Principles {#intro}
+
+Information is power. It can be used to predict and to influence people, as well as to design online
+spaces that control people's behaviour. The collection and [=processing=] of information in greater
+volume, with greater precision and reliability, with increasing interoperability across a greater
+variety of data types, and at increasing speed is leading to an unprecedented concentration of power
+that threatens private and public liberties.
+
+What's more, automation and the increasing computerisation of all aspects of our lives
+both increase the power of information and decrease the cost of a number of intrusive
+behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
+the victim.
+
+These asymmetries of information and of automation create commanding asymmetries of power. It is not
```

Comment:
I'm happy to pick a different word, but the reason I wanted one is because there are always asymmetries of power, but we only care when they are of a certain size. Maybe I slept better last night, maybe you come across better in meetings because you have better bandwidth — these are asymmetries that can give some power, but they're not in the same class as "I know everything you did over the past fifteen years."